frontend/apps/marketing/src/providers/statsig/client.ts

```diff
@@ -1,11 +1,28 @@
 import {useClientBootstrapInit} from '@statsig/react-bindings';
+import {getCookie} from 'cookies-next/client';
+import {v4 as uuidv4} from 'uuid';
 
 import {Stage} from '@/config/stage';
 import plugins from '@/providers/statsig/plugins';
 
+function getStatsigStableId() {
+  let stableId = getCookie('statsig_stable_id');
+
+  if (!stableId) {
+    stableId = uuidv4();
+  }
+
+  return stableId;
```

Comment:
Should this be set as the value of the `statsig_stable_id` cookie?

Reply:
Agreed, something like this? https://github.com/code-dot-org/code-dot-org/blob/1c336e982e8274441ed5f47d587fa2c8e2e94e9c/apps/src/metrics/statsigHelpers.js#L15-L25

Reply:
Oh shoot! I was literally looking at that and forgot it. I updated to include this logic.

I also tossed in the feature where this cookie shouldn't be set if performance cookies are disabled.